### PR TITLE
podvm: Pin attestation-agent

### PR DESCRIPTION
**Pull request by stevenhorsman**
- Pin attestaton-agent to v0.4.0 to avoid instability

Fixes: #697

### CODE REVIEW

**Review by bpradipt — APPROVED** [review]
/lgtm
We need a version.yaml asap ;)

File: podvm/Makefile.inc [hunk 1]
```diff
@@ -10,6 +10,7 @@
 UBUNTU_RELEASE     = focal
 SKOPEO_VERSION     = 1.5.0
 UMOCI_VERSION      = 0.4.7
+AA_VERSION		   = v0.4.0
```
Comment (jtumber-ibm) [inline podvm/Makefile.inc:13]: nit: space/tab consistency

Comment (stevenhorsman) [reply podvm/Makefile.inc]: Thanks - it should be updated now

**Comment by stevenhorsman** [comment]
For full disclosure I've run an image created from this code through some tests, but they don't utilise the attestation-agent, so I can't 100% guarantee that backwards compatibility is fully maintained.

**Review by jtumber-ibm — APPROVED** [review]

**Pull request merged**